new_platforms/docs/visualstudio_dev.md

```diff
@@ -0,0 +1,109 @@
+Developing your own enclave using Visual Studio
+=============
+
+## Using Open Enclave from Windows
+
+This section covers how to use this SDK to develop your own enclave that can run
+on both SGX and OP-TEE, and use it from your own application.
+
+**Prerequisites:**
+1. Install the Open Enclave Visual Studio Extension
+2. To develop for SGX, install the [Prerequisites for SGX](win_sgx_dev.md)
+2. To develop for OP-TEE, install the [Prerequisites for OP-TEE](ta_debugging_wsl.md)
+
+**To create your own enclave:**
+1. In Visual Studio, add a new Visual C++ "Open Enclave TEE Project".
+2. Edit the _YourProjectName_.edl file. Define any trusted APIs (called "ECALLs")
+you want to call from your application in the trusted{} section,
+and in the untrusted{} section, define any application APIs (called "OCALLs")
+that you want to call from your enclave. Definitions must be done using the
```

Comment:
Perhaps: "Definitions must be *described* using the [...]"

Reply:
Done